shared/AppInsightsCore/Tests/Selenium/ApplicationInsightsCore.Tests.ts

```diff
@@ -598,38 +597,25 @@ export class ApplicationInsightsCoreTests extends TestClass {
             test: () => {
                 let map = {};
 
-                let mwcRandCalled = 0;
-                let randCalled = 0;
+                // Check that mwcRandom is bing called (relies on the mwc implementation from the default seed)
+                mwcRandomSeed(1);
+                Assert.notEqual(722346555, random32(), "Make sure that the mwcRandom was not being called - step 1");
+                Assert.notEqual(3284929732, random32(), "Make sure that the mwcRandom was not being called - step2");
 
-                // Using manual spies as sinon spy is failing to correctly restore the function
-                let orgMwcRandom = CoreUtils.mwcRandom32;
-                CoreUtils.mwcRandom32 = function() {
```

Comment:
Generally, I tried to not change any of the Tests so that they continue to use the CoreUtils versions (to help ensure no-one is broken). However, in this case because it mocks the values but the internal code no longer uses these references these tests had to change to continue passing.